src/main/resources/modules/lung_cancer.json

```diff
@@ -1501,7 +1501,7 @@
       "type": "ImagingStudy",
       "procedure_code": {
         "system": "SNOMED-CT",
-        "code": 16335031000119103,
+        "code": 16335031000119104,
```

Comment:
Did you mean to include these code changes in this pull request?

Reply:
No... and I don't have any idea how this happened.

Reply:
Double precision issue maybe?

Reply:
@dehall yes...

```
Welcome to Node.js v14.8.0.
Type ".help" for more information.
> foo = '{"code":16335031000119103}'
'{"code":16335031000119103}'
> parsed = JSON.parse(foo)
{ code: 16335031000119104 }
> Number('16335031000119103')
16335031000119104
> Number.isSafeInteger(16335031000119103)
false
> 
```